prometheus_reporter.cc

```diff
@@ -0,0 +1,84 @@
+#include "prometheus_reporter.hpp"
+
+#include <string>
+#include <iostream>
+
+#include <prometheus/gauge.h>
+#include <prometheus/registry.h>
+#include <prometheus/gateway.h>
+#include <prometheus/counter.h>
+
+PrometheusReporter::PrometheusReporter(std::string host, std::string port,
+                                       std::string scenario, std::string api, std::string location, std::string universe, std::string grpc_version, std::string gcs_client_version) : registry_(std::make_shared<prometheus::Registry>()),
+                                                                                                                         gateway_(host, port, "gcs-perf-prober", 
```

Comment:
The formatting here hurts my eyes.  Please consider a `.clang-format` file and using `clang-format`.